cypress/e2e/main.spec.js

```diff
@@ -189,7 +189,7 @@ context('Window', () => {
 
     it('should cache pages from absolute URLs', () => {
         cy.window().then(window => {
-            window._swup.loadPage({ url: 'http://localhost:8080/page2/' });
+            window._swup.loadPage({ url: 'http://localhost:8274/page2/' });
```

Comment:
Could we somehow get this as a variable from cypress.config.js? Might come in handy if we keep referencing absolute URLs in tests. Or maybe that's overkill, seeing how it's hardcoded all over the place anyway? Your judgement :)

Reply:
Let's keep it dumb for now, ok? Changing the port is a simple search-replace, after all :)

Reply:
Okay :)